internal/constants/constants.go

```diff
@@ -129,6 +129,9 @@ const DisableAnalyticsEnvVarName = "ACTIVESTATE_CLI_DISABLE_ANALYTICS"
 // OptinUnstableEnvVarName is used to instruct State Tool to opt-in to unstable features
 const OptinUnstableEnvVarName = "ACTIVESTATE_OPTIN_UNSTABLE"
 
+// ServiceSockPath overrides the default socket path root diriectory used by the state service
+const ServiceSockPath = "ACTIVESTATE_SVC_SOCK"
```

Comment:
Nit: Please consider renaming this ServiceSockDir to emphasize that it's not the path to the sock file itself.